resources/styles/components/task/index.less

```diff
@@ -0,0 +1,35 @@
+.fixed-bar() {
+  left: 0;
+  position: fixed;
+  width: 100%;
+  background-color: fadeout(@tutor-white, 20%);
+}
+
+
+.task-view {
+  padding-top: 2 * @tutor-navbar-height;
+  padding-bottom: @tutor-navbar-height;
+
+  .panel-header {
```

Comment:
Should this still be a `panel-header` since it is always pinned? I'm worried that we are overusing bootstrap terms and changing the semantics of what these elements are.